tests/test_inspector.py

```diff
@@ -103,6 +103,8 @@ def setUpClass(cls):
         # Last file to be left without violations
 
         cls.nwbfile_paths = [str(cls.tempdir / f"testing{j}.nwb") for j in range(num_nwbfiles)]
+        nwbfiles.append(make_minimal_nwbfile())
```

Comment:
A minimal NWB file would have no problems with it though

Reply:
Maybe bump the number of files to 4, do one of the common violations to the last one, and add `._` to the autogenerated `nwbfile_path` of the last one?

Reply:
how about this?

Reply:
Closer - it will have to be an additional one tacked onto the end though, not index 2 (since that already exists and has specific tests for it)

Either that or a simpler standalone unit test for just this particular behavior

Reply:
ok now the sidecar test file is the last one in the list and the no-error file is back to being the third